go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20191115003340-16619cd27fa5
 	github.com/operator-framework/operator-marketplace v0.0.0-20191105191618-530c85d41ce7
 	github.com/operator-framework/operator-sdk v0.15.1
+	github.com/pkg/errors v0.8.1
```

Comment:
There's been an effort by Ger to remove the usage of `pkg.Errors` and to use instead the `fmt.Errors` package instead, so would be good to stick to one for consistency 
https://github.com/integr8ly/integreatly-operator/pull/390/files

Reply:
thanks Kevin, I'll remove this